client/src/style/scss/theme/blue.scss

```diff
@@ -79,6 +79,8 @@ $state-success-text: theme-color-level("success", $alert-color-level);
 $state-success-bg: theme-color-level("success", $alert-bg-level);
 $state-success-border: theme-color-level("success", $alert-border-level);
 
+$state-secondary-text: theme-color-level("secondary", $alert-color-level);
```

Comment:
Is this used anywhere?

Reply:
I added it for the text color for queued states. I didn't see anything else that seemed to match up with my expectations. Is there something else in this file that you think is a good match?